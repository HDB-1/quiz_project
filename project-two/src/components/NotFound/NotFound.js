import React from 'react'

export default function NotFound() {
    return (
        <div>
            <h2>Page not found: please use the link above to return home</h2>
        </div>
    )
}
