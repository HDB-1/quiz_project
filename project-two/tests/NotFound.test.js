import React from 'react'
import {shallow} from 'enzyme'
import NotFound from '../src/components/NotFound/NotFound'

describe('NotFound', () => {
    it('Should render an h2', () => {
        let wrapper = shallow(<NotFound />)
        expect(wrapper.find('h2').length).toEqual(1);
    })
    it('Should render correctly', () => {
        let wrapper = shallow(<NotFound />)
        expect(wrapper).toMatchSnapshot();
    })
})