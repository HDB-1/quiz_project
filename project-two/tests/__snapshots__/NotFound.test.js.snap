// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Should render correctly 1`] = `
<div>
  <h2>
    Page not found: please use the link above to return home
  </h2>
</div>
`;
